testsuite/MDAnalysisTests/topology/test_dlpoly.py

```diff
@@ -43,10 +43,16 @@ def test_creates_universe(self, filename):
         u = mda.Universe(filename, topology_format=self.format)
         assert isinstance(u, mda.Universe)
 
+    def test_guessed_attributes(self, filename):
+        u = mda.Universe(filename, topology_format=self.format)
+        for attr in self.guessed_attrs:
+            assert hasattr(u.atoms, attr)
+
 
 class DLPBase2(DLPUniverse):
     expected_attrs = ['ids', 'names']
-    guessed_attrs = ['types', 'masses']
+    guessed_attrs = ['masses', 'types']
+
     expected_n_atoms = 216
     expected_n_residues = 1
     expected_n_segments = 1
```

Comment:
```suggestion
    expected_n_segments = 1
    
    def test_masses(self, top):
        assert_equal(top.masses.values[0], 39.102)
        assert_equal(top.masses.values[4], 35.45)
    
    def test_types(self, top):
        assert top.types.values[0] == 'K'
        assert top.types.values[4] == 'CL'
```

Suggested additional test to compare to real values